tox.ini

```diff
@@ -79,17 +76,14 @@ commands =
 
 [testenv:travis]
 basepython = python3.6
-passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY GANACHE_BINARY
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY GANACHE_BINARY BRANCH
 deps =
 	pip-tools
 	coverage
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-    pip-sync requirements/base.txt requirements/tests.txt
+    pip-sync requirements/base.txt requirements/tests.txt requirements/pandapower.txt
```

Comment:
Correct me if Im wrong, didn't we skipped it in first place due to `pandapower` incompatability with `pypy` ?

Reply:
Yes, but we do not use pypy when running tests AFAIK. Pandapower is needed for the tests due to its integration test, that's why it is included here.